docker/docker-build-and-push.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+
+sh docker-build.sh $@
+if [[ "$#" -gt 0 ]]
+then
+  DOCKER_TAG=$1
+else
+  DOCKER_TAG=test
+fi
+echo "Trying to push docker image to ${DOCKER_TAG}"
+docker push ${DOCKER_TAG}
```

Comment:
Which repository this will push? As long as this won't push to the official apache/pinot for docker hub, I'm fine with this pr.